### PR TITLE
Updated to PyCon 2016.

### PR DESCRIPTION
**Pull request by helioloureiro**
- subscription link needs update.
 - 2015 site link doesn't exist yet.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,15 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="author" content="Python Sverige">
     <meta name="keywords" content="python pycon django conference meetup">
-    <meta name="description" content="PyCon Sweden 2015 is taking place in Stockholm on May 12-13, 2015. Join us and be part of the second national Python conference in Sweden.">
-    <meta property="og:title" content="PyCon Sweden 2015 - May 12-13 in Stockholm" />
-    <meta property="og:site_name" content="PyCon Sweden 2015 - May 12-13 in Stockholm" />
+    <meta name="description" content="PyCon Sweden 2016 is taking place in Stockholm on May 12-13, 2016. Join us and be part of the second national Python conference in Sweden.">
```
Comment (nailor) [inline index.html]: Dates are wrong, they should be May 9 - 10

File: index.html [hunk 2]
```diff
@@ -27,7 +27,7 @@
     <div class="container">
       <div class="header">
         <ul class="nav nav-pills pull-right">
-          <li><a href="http://pyconse.tumblr.com">Blog</a></li>
+          <li><a href="#cfp">Call For Proposals</a></li>
           <li><a href="sponsorship.html">Interested in sponsoring?</a></li>
```
Comment (nailor) [inline index.html]: Maybe we should remove the sponsorship link until I've fixed some levels for this year

**Comment by nailor** [comment]
Added some notes, but we should also copy the 2015 page under 2015 folder (with some info, like sponsorship levels, removed)

File: index.html [hunk 3]
```diff
@@ -37,259 +37,66 @@
...
           <div class="col-sm-12">
             <h2>It's time for PyCon Sweden again!</h2>
             <p>
-              Following last years success we are proud to announce the 2015 edition of PyCon Sweden. 
+              Following last years success we are proud to announce the 2016 edition of PyCon Sweden. 
               Just like last year it will be two full days of talks, activities and fun!
             </p>
-            <h3>Where and when does the conference take place?</h3>
-            <p>
-              PyCon Sweden 2015 will take place in Stockholm on May 12-13 at the <a href="http://www3.hilton.com/en/hotels/sweden/hilton-stockholm-slussen-STOSLHI/event/index.html" target="_new">Hilton Stockholm Slussen</a> hotel.
```
Comment (nailor) [inline index.html]: 2016 May 9-10 at the same Hilton

**Comment by frecon** [comment]
PyCon 2016 \o/

File: index.html (hunk 1 above)
Comment (helioloureiro) [inline index.html]: Fixed!

File: index.html (hunk 2 above)
Comment (helioloureiro) [inline index.html]: I commented everything regarding sponsorship.

**Comment by helioloureiro** [comment]
Included changes in other commit.

**Comment by frvi** [comment]
Should we maybe have a link somewhere to our youtube channel as well?

**Comment by haard** [comment]
Created https://github.com/python-sverige/python-sverige.github.io/pull/41 to enable collaborative work

**Comment by haard** [comment]
Closing this in favor of the branch version which includes all changes from this PR as well

**Pull request closed without merging**